projects/ids-enterprise-ng/src/lib/version/version-initializer.service.spec.ts

```diff
@@ -6,12 +6,12 @@ describe('Soho Version Initialiser Service', () => {
   beforeEach(() => TestBed.configureTestingModule({
     imports: [
     ],
-     providers: [
+    providers: [
     ]
   }));
 
   it('should be created', () => {
-    const service: SohoVersionInitializerService = TestBed.get(SohoVersionInitializerService);
+    const service: SohoVersionInitializerService = TestBed.inject(SohoVersionInitializerService);
```

Comment:
So in all tests it's inject instead of get?

Reply:
Yep - get is deprecated.